openqaoa/optimizers/training_vqa.py

```diff
@@ -232,8 +232,8 @@ def optimize_this(self, x):
         if hasattr(self.vqa, 'job_id'):
             log_dict.update({'job_ids': self.vqa.job_id})
```

Comment:
Probably we should devise ways to test the loggers better. I can see that quite a few lines are not covered by the tests.

However, this may be part of a separate PR?

Reply:
This specific line requires a QPU test to have checked. so are the log_with_backend methods, which are only available in AWS Jobs

Reply:
Yes, I know. I am wondering if it would make sense to have a mock test for this ... but anyway, it is out of the scope of this PR. Sorry!